gamemodes/terrortown/gamemode/client/cl_lang.lua

```diff
@@ -69,7 +69,7 @@ function LANG.AddToLanguage(langName, stringName, stringText)
 	langName = LANG.GetNameFromAlias(langName)
 
 	if not LANG.IsLanguage(langName) then
-		ErrorNoHalt(Format("Failed to add '%s' to language '%s': language does not exist.\n", tostring(stringName), tostring(langName)))
+		ErrorNoHaltWithStack(Format("Failed to add '%s' to language '%s': language does not exist.\n", tostring(stringName), tostring(langName)))
```

Comment:
as I said in my last review, my console is already spammed with lang deprecation warning. If we add t more lines to each warning I think the console becomes unusable, don't you think?

Reply:
well this comment is tied to the wrong line, sorry about that. This specific change here is good.

But in the same file is a deprecation warning. I'm talking about the one in line 223, the rest seems fine

Reply:
the problem is that this error does not indicate the oirigin of the problem. even if i wanted to fix this, i would need to unpack every addon i have in order to locate the problematic language file, and it's so far removed from the source file directory that it cannot be associated _without_ a stack trace

Reply:
I absolutely understand that. But 99% of the people only want to play TTT2 and will be pretty confused when there are errors on every mapchange that they can't fix

Reply:
I reverted this, @Histalek and I talked about a better solution in our debug library that can be done in a coming PR so that we can merge this one!